packages/core/contracts/financial-templates/common/financial-product-libraries/KpiOptionsFinancialProductLibrary.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity ^0.6.0;
+pragma experimental ABIEncoderV2;
+import "./FinancialProductLibrary.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "../../../common/implementation/Lockable.sol";
+
+/**
+ * @title KPI Options Financial Product Library
+ * @notice Adds custom tranformation logic to modify the price and collateral requirement behavior of the expiring multi party contract.
+ * If a price request is made pre-expiry, the price should always be set to 2 and the collateral requirement should be set to 1.
+ * Post-expiry, the collateral requirement is left as 1 and the price is left unchanged.
+ */
+contract KpiOptionsFinancialProductLibrary is FinancialProductLibrary, Ownable, Lockable {
```

Comment:
nit: can we drop ownable and lockable? I don't think they're used in this lib.

Reply:
Isn't lockable needed for `nonReentrantView()`?

Reply:
Or is `nonReentrantView()` also not needed because there is no `nonReentrant()` method?

Reply:
I think we can drop nonReentrantView yes. It doesn't make external calls

Reply:
Hmm technically this makes external contract calls but we also trust these contracts. Hard to say. Is the main benefit to removing Ownable and Lockable byte code and gas? @mrice32

Reply:
Removed Ownable since it seemed completely unused, and left Lockable for now.